roles/sro_deploy_art_bundle/files/builder_namespace.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ci-artifacts
+  labels:
+    app: ci-artifacts
+spec: {}
```

Comment:
new line at end of file

(I'll add a Github Action to enforce that soon ;-))

Reply:
Done